test/parallel/test-http-agent-uninitialized.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const common = require('../common');
+const http = require('http');
+const net = require('net');
+
+const agent = new http.Agent({
+  keepAlive: true,
+});
+const socket = new net.Socket();
```

Comment:
@trevnorris Re your comment https://github.com/nodejs/node/issues/13045#issuecomment-302270303
Could you add a test case for a `Socket` with a fake `_handle`?